src/docfx/config/ops/OpsConfigAdapter.cs

```diff
@@ -20,12 +20,6 @@ internal class OpsConfigAdapter : IDisposable
         private const string MetadataSchemaApi = "https://ops/metadataschema/";
         private const string MarkdownValidationRulesApi = "https://ops/markdownvalidationrules/";
 
-        private static readonly string s_opsToken = Environment.GetEnvironmentVariable("DOCS_OPS_TOKEN");
```

Comment:
Does this require docs-pipeline to set the http header for OPS urls?
Also remove https://ceapex.visualstudio.com/Engineering/_git/Docs.Build?path=%2Fsrc%2Fdocs-pipeline%2FDocsPipeline.cs&version=GBdevelop&line=56&lineEnd=57&lineStartColumn=1&lineEndColumn=1&lineStyle=plain

Reply:
yep, need a new PR